manager/filters.py

```diff
@@ -619,6 +619,40 @@ def _filter_system_by_host_type(query, args, _kwargs):
     return query
 
 
+def _filter_cve_by_affecting_host_type(query, args, kwargs):
+    """
+    Filters list of CVEs based on the fact whether CVE applies at least to one system (either edge or rpmdnf) or not.
+    Expects count subquery associated with system_platform.
+
+    Args:
+        query (object): Query object to apply filters to
+        args (dict): Query arguemnts
+        kwargs (dict): subquery object
+
+    Returns:
+        object: Modified query with affecting filter applied
+    """
+    if args.get("is_cached"):
```

Comment:
Shouldn't this check `kwargs["is_cached"]`?

Reply:
good catch